code/game/machinery/computer/communications.dm

```diff
@@ -115,9 +113,9 @@
 						playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 50, 0)
 						//Only notify people if an actual change happened
 						var/security_level = get_security_level()
-						log_game("[key_name(usr)] has changed the security level to [security_level].")
-						message_admins("[key_name_admin(usr)] has changed the security level to [security_level].")
-						deadchat_broadcast("<span class='deadsay'><span class='name'>[usr.real_name]</span> has changed the security level to [security_level] at <span class='name'>[area_name]</span>.</span>", usr)
+						log_game("[key_name(usr)] has changed the security level to [security_level] with [src] at [AREACOORD(usr)].")
+						message_admins("[ADMIN_LOOKUPFLW(usr)] has changed the security level to [security_level] with [src] at [AREACOORD(usr)].")
+						deadchat_broadcast("<span class='deadsay'><span class='name'>[usr.real_name]</span> has changed the security level to [security_level] with [src] at <span class='name'>[AREACOORD(usr)]</span>.</span>", usr)

```

Comment:
Deadchat doesn't really benefit from coordinates, just the area name should be fine here. Ditto the other `deadchat_broadcast`s in this file.